### PR TITLE
[css-variables] Test attribute changes that affect generated content defined via CSS variables.

### PR DESCRIPTION
**Pull request by emilio**
This adds a test for:

  https://bugs.chromium.org/p/chromium/issues/detail?id=772316

Testcase passes in Gecko and WebKit, not in Blink. Haven't tested Edge.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19251)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/284462309)

**Comment by Ms2ger** [comment]
Is there a particular reason to use variables at all in the reference?

**Comment by emilio** [comment]
Nope, but it doesn't, right?

File: css/css-variables-1/variable-generated-content-dynamic-001-ref.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Test Reference</title>
+<style>
+div::before {
+  content: attr(data-foo);
+}
+</style>
+<div data-foo="PASS"></div>
```
Comment (gsnedders) [inline css/css-variables-1/variable-generated-content-dynamic-001-ref.html]: Does `<div>PASS</div>` not suffice here?

Comment (emilio) [reply css/css-variables-1/variable-generated-content-dynamic-001-ref.html]: It probably should, but blink has had problems with reftests that exercised text-shaping across different node boundaries (see https://github.com/w3c/csswg-test/pull/1143). That being said, that is being fixed, so I'll update, I guess.

Comment (emilio) [reply css/css-variables-1/variable-generated-content-dynamic-001-ref.html]: Just did that.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**